### PR TITLE
Move attributes to separate project

### PR DESCRIPTION
**Pull request by cd21h**

### CODE REVIEW

File: src/MessagePack.Annotations/MessagePack.Annotations.csproj [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.6;netstandard2.0;net45;net47</TargetFrameworks>
```
Comment (AArnott) [inline src/MessagePack.Annotations/MessagePack.Annotations.csproj]: We don't need so many targets here. I suspect `netstandard1.0` is adequate by itself.

**Review by AArnott — APPROVED** [review]
Thanks for preparing this. Can you make one change?

**Comment by AArnott** [comment]
@neuecc any thoughts? I imagine this will end up getting merged into the root project soon.

File: src/MessagePack.Annotations/MessagePack.Annotations.csproj (hunk 1 above)
Comment (cd21h) [reply src/MessagePack.Annotations/MessagePack.Annotations.csproj]: Sure, will do today

**Comment by neuecc** [comment]
very good.

**Comment by cd21h** [comment]
@aarnott can you publish it to nuget?

**Comment by AArnott** [comment]
Not yet, @shatl. First off, this is a fork off the main project, and the main project is where the MessagePack nuget package publishes from. We'll get this PR merged into that root project eventually. But at the moment, being that your PR targets master, that's slated for v2, which is under heavy development and not ready for a release yet.

If you want to get this out more quickly, you could target this project's `v1.8` branch with a copy of these changes. To avoid a binary breaking change, you'd also need to add `TypeForwardedTo` attributes to the original assembly for each type you moved.

**Pull request merged**